Source/WebCore/css/parser/CSSPropertyParserConsumer+LengthPercentage.cpp

```diff
@@ -177,18 +177,6 @@ RefPtr<CSSPrimitiveValue> consumeLengthPercentage(CSSParserTokenRange& range, co
         .unitlessZero = unitlessZero
     };
 
-    // FIXME: 'anchor' should be handled and exposed as a calc() function type.
-    auto rangeCopy = range;
-    if (rangeCopy.peek().functionId() == CSSValueAnchor) {
-        if (options.anchorPolicy == AnchorPolicy::Allow) {
-            if (auto anchor = consumeAnchor(rangeCopy, context)) {
```

Comment:
Should we remove `consumeAnchor` too, now that it's not used? (`CSSPropertyParsersHelpers::consumeAnchor`)